modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/item/selector/ContentDashboardItemSubtypeItemSelectorView.java

```diff
@@ -299,6 +300,14 @@ private void _populateContentDashboardItemTypesJSONArray(
 						return infoItemClassDetailsLabelInfoLocalizedValue.
 							getValue(themeDisplay.getLocale());
 					}
+				).put(
+					"selected",
+					checkedContentDashboardItemSubtypeInfoItemReferences.
+						contains(
+							new InfoItemReference(
+								BlogsEntry.class.getName(),
```

Comment:
Hey @Stephy6 !!

We can make Content Dashboard dependes on Blogs :(

It should be a API.

Thanks!!

Reply:
We changed it so that it changes the className of the info item reference, and decoupled it